cypress/e2e/settings/hidden-rr-migration.spec.ts

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
I think it would make sense to make this into `settings-migrations.spec.ts` and put the hidden RRs into a `describe`. This way new settings migrations have a place to go

(**this is more of a personal preference and it is not a blocker**)